Assets/ConnectApp/Components/CustomTabBar.cs

```diff
@@ -41,22 +43,26 @@ public class CustomTabBar : StatefulWidget {
     public class CustomTabBarState : State<CustomTabBar> {
         PageController _pageController;
         int _selectedIndex;
-
-        const int KTabBarHeight = 49;
+        float _bottomPadding;
 
         public override void initState() {
             base.initState();
             this._selectedIndex = 0;
+            this._bottomPadding = 0;
             this._pageController = new PageController(this._selectedIndex);
         }
 
         public override Widget build(BuildContext context) {
+            if (this._bottomPadding != MediaQuery.of(context).padding.bottom &&
+                Application.platform != RuntimePlatform.Android) {
+                this._bottomPadding = MediaQuery.of(context).padding.bottom;
+                this.setState(() => { });
```

Comment:
(1) this code block is shared in many widgets, anyway to make it something global ?
(2) why call setState() in build ? seems unnecessary?  any async operation involved here?

Reply:
(1) it is a good idea, but the code needs `context` get padding, you have any idea.
(2) actually, I don't know the sync code needn't call setState(). I will fix it. thanks.

Reply:
（1）maybe pass the context into a static function, e.g.,  public static float GetCustomBottomPadding(BuildContext context),  inside a static Utils class ?

  (2)  no prob. MediaQuery.of(context).padding is sync, just remove this setState() and give it a try? it may work

Reply:
(1)  I find many places used `MediaQuery.of(context).padding`, but not only bottom, I want to delay to fix it. I already record it.
(2) fixed.